indico/modules/events/surveys/templates/submission_preview.html

```diff
@@ -0,0 +1,36 @@
+{% extends 'events/surveys/manage_survey_base.html' %}
+{% from 'forms/_form.html' import form_row %}
+
+{% set survey = submission.survey %}
+
+{% block content %}
+    <div class="i-box-content">
+        <ul class="group-list no-content-before submission-preview">
+            {% for answer in submission.answers %}
+                <li>
+                    <div class="i-form survey-answer">
+                        {{ form_row(form['question_{}'.format(answer.question.id)], orientation='horizontal') }}
+                    </div>
+                </li>
+            {% endfor %}
+            <li>
+                <div class="i-form">
+                    <div class="form-group horizontal">
+                        <div class="form-label form-block"></div>
+                        <div class="form-field form-block">
+                            <a href="{{ url_for('.manage_survey', submission.survey) }}" class="i-button big" data-button-back>
+                                {%- trans %}Back{% endtrans -%}
+                            </a>
+                        </div>
+                    </div>
+                </div>
+            </li>
+        </ul>
+    </div>
+
+    <script>
+        $('.survey-answer :radio:not(:checked), .survey-answer :checkbox').prop('disabled', true);
+        $('.survey-answer input[type=text]').prop('readonly', true);
```

Comment:
However, it doesn't look good in case of BooleanFields. You can't clearly see the answer on disabled `BooleanField`. I think some css rules for it would do the trick.